index.js

```diff
@@ -26,7 +26,7 @@ export default class CrossfadeImage extends Component {
             if (!this.timeout) clearTimeout(this.timeout);
             this.timeout = setTimeout(
               () => this.setState({ bottomOpacity: 0 }),
-              10
+              50
```

Comment:
Thanks a lot. You're spot on. 10ms sometimes leaves out the fade on Firefox.
However we might not need up to 50, I tested at 15ms and it didn't fail once so I would suggest 20ms.
Just make another commit if you agree. Let me know if you have any other idea.
Cheers.